src/assets/style.css

```diff
@@ -0,0 +1,23 @@
+.title {
```

Comment:
src/style.css and src/assets/style.css might be confusing

Reply:
src/assets/style.css removed in [c957e31](https://github.com/InsaLan/frontend-insalan.fr/pull/40/commits/c957e3197aa63677e2c639b39ac1f1cd63997b64)